### ISSUE TITLE
No network connectivity issues mitigation.

### ISSUE THREAD

**Issue by matt-kita** [ehdinayan/Cloud-exercise2#1]
isodeb.sh:
curl -O https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/MD5SUMS

If cdimage.debian.org server is temporarily unavailable, the script just stops at this point, and "hangs" indefinitely without any outputs. Additionally, any existing MD5SUMS file present in current working directory will be erased and overwritten with this operation's outcome - this is unacceptable - checksum for new file should be at most stored in memory, prior to any new files being downloaded.

**Comment by ehdinayan** [comment]
I got that solved Mateusz, don't worry thank you. It is ok to me overwrite old md5sum files that will not match current iso versions.
Also most easy way to avoid this "issue" is to have a dir where storing those files, wich I am pretty sure won't be the same as where executing the program. As simple as that.

**Comment by matt-kita** [comment]
I haven't seen any viable solution yet :)

**Comment by ehdinayan** [comment]
I don't want the program search for other MD5SUMS files in dir in order to move them so won't be overwritten, if you are reffering to that, Matt.

**Comment by matt-kita** [comment]
No, this issue is mainly about mitigating any network connectivity problems.

I already got your approach regarding any MD5SUMS file replacement... although I still strongly disagree with it :)
But it's your CPU cycles (for recalculating MD5s), and your disk I/O (for saving/replacing MD5SUMS files), so wasting resources shouldn't really be my concern - I'd just do it differently, not to waste any of them unnecessarily :)

**Comment by ehdinayan** [comment]
I cannot be aware of cpu cycles while writing program as you Know Matt, there's nothing unnusual on that given my knowledge you know...I just started this year! But when program is ok finished to me will be ok if you want to share any improvements man of course! I hope so! program tunning jajaj. Much cheaper and as interesting as car or motorbike tunning you know?
